src/wallet3/cli-wallet/oxen_wallet_cli/walletcli.py

```diff
@@ -98,12 +112,67 @@ def address():
     click.echo("Address: {}".format("TODO sean get the address here"))
 
 @walletcli.command()
-def get_balance():
-    click.echo("Balance: {}".format(context.wallet.get_balance()))
+def balance():
+    if context.wallet is None:
+        click.echo("Wallet not loaded")
+        return
+    click.echo("Balance: {:.2f} Oxen".format(context.wallet.get_balance()/1e9))
+
+@walletcli.command()
+def unlocked_balance():
+    if context.wallet is None:
+        click.echo("Wallet not loaded")
+        return
+    click.echo("Unlocked Balance: {:.2f} Oxen".format(context.wallet.get_unlocked_balance()/1e9))
+
+@walletcli.command()
+def height():
+    height_future = context.rpc_future("rpc.get_height");
+    height = height_future.get();
+    click.echo("Height: {}".format(height))
+
+@walletcli.command()
+def transfer():
+    address = click.prompt("Enter the destination wallet address", default="").strip()
+    amount = click.prompt("Enter the amount in oxen to be sent to {}".format(address), default=0.0)
+    if address == "" or amount == 0.0:
+        click.prompt("Invalid address/amount entered")
+        return
+    amount_in_atomic_units = round(amount * 10e9, 0);
+    destination = {"address": address, "amount": amount_in_atomic_units}
+    transfer_params = {"destinations": [destination]}
+    transfer_future = context.rpc_future("restricted.transfer", args=transfer_params);
+    transfer_response = transfer_future.get();
+    click.echo("Transfer Response: {}".format(transfer_response))
+
+@walletcli.command()
+def ons_buy_mapping():
+    ons_type = click.prompt("What type of mapping would you like", type=click.Choice(['session', 'wallet', 'lokinet', 'lokinet_2years', 'lokinet_5years', 'lokinet_10years']), default="session").strip()
```

Comment:
I think this could be usefully split into multiple options: first session/wallet/lokinet, and then if you choose lokinet, we ask the length.